dev/reference/performance/statistics.md

```diff
@@ -15,24 +15,81 @@ Based on the statistics, the TiDB optimizer chooses the most efficient query exe
 
 ### Manual collection
 
-You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
+You can run the `ANALYZE` statement to collect statistics.
 
-Syntax:
+#### Full collection
+
+> **Note:**
+>
+> `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
```

Comment:
- Use `might` instead of `may` because `may` has multiple meanings.

```suggestion
> The execution time of `ANALYZE TABLE` in TiDB is longer than that in MySQL or InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
```